ui/app/img/stickers_icon_open.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
BTW, if you just need to change the color of an icon, there is a ColorOverlay in QML that enables to do that. Eg: https://github.com/status-im/nim-status-client/blob/master/ui/app/AppLayouts/Wallet/LeftTab.qml#L109-L112